src/plugins/enzymes/src/FindEnzymesDialog.cpp

```diff
@@ -1021,6 +1042,38 @@ QString EnzymeTreeItem::generateEnzymeTooltip() const {
     return TOOLTIP_TAG.arg(forwardTooltip).arg(reverseTooltip);
 }
 
+QString EnzymeTreeItem::getTypeInfo() const {
+    auto type = text(Column::Type);
+    QString result;
+    static constexpr const char* UNEXPECTED_TYPE_ERROR = QT_TR_NOOP("Unexpected enzyme type: %1");
+    if (type == "M") {
+        result = tr("An orphan methylase,<br>not associated with a restriction enzyme or specificity subunit");
+
+    } else if (type.size() == 2) {
+        if (type == "IE") {
+            result = tr("An intron-encoded (homing) endonuclease");
+        } else if (type.startsWith("R")) {
+            result = tr("The restriction enzyme of the %1 type").arg(type.back());
+        } else if (type.startsWith("M")) {
+            result = tr("The methylase of the %1 type").arg(type.back());
+        } else {
+            coreLog.details(tr(UNEXPECTED_TYPE_ERROR).arg(type));
+        }
+    } else if (type.size() == 3) {
+        if (type.startsWith("R") && type.endsWith("*")) {
+            result = tr("The restriction enzyme of the %1 type,<br>but only recognizes the sequence when it is methylated").arg(type.at(1));
+        } else if (type.startsWith("RM")) {
+            result = tr("The enzyme of the %1 type, which acts as both -<br>a restriction enzyme and a methylase").arg(type.back());
+        } else {
+            coreLog.details(tr(UNEXPECTED_TYPE_ERROR).arg(type));
+        }
+    } else {
+        coreLog.details(tr(UNEXPECTED_TYPE_ERROR).arg(type));
```

Comment:
Remove logging about our parsing issues completely. If you think that our parsing failure is important to user just show this in the view.

Reply:
OK